### ISSUE TITLE
npm Ion defaultAccessToken Error

### ISSUE THREAD

**Issue by wenzhihao123** [CesiumGS/cesium#11290]
Hello，i use cesium 1.105.1 in my npm+rollup project，then i use my npm project in my business project，i dispose business project by coping files and set CESIUM_BASE_URL on window object,
```
node_modules/cesium/Build/Cesium/Assets
node_modules/cesium/Build/Cesium/ThirdParty
node_modules/cesium/Build/Cesium/Widgets
node_modules/cesium/Build/Cesium/Workers
```
```
(window as any).CESIUM_BASE_URL = 'cesium/'
```
it's work fine，and i see the earth，but when i dispose Ion token，there have  some problems.

i have set this code, just like this:
```
Cesium.Ion.defaultAccessToken = 'my token'
Cesium.Camera.DEFAULT_VIEW_RECTANGLE = Cesium.Rectangle.fromDegrees(90, -20, 110, 90)
```
but i still get the tips in the page:
`
This application is using Cesium's default ion access token. Please assign Cesium.Ion.defaultAccessToken with an access token from your ion account before making any Cesium API calls. You can sign up for a free ion account at https://cesium.com/. • [Upgrade for commercial use.](https://cesium.com/pricing/)Data attribution
`
and the DEFAULT_VIEW_RECTANGLE did't work also.

Here is my rollup.config
```
import typescript from 'rollup-plugin-typescript2'
import postcss from 'rollup-plugin-postcss'
import image from '@rollup/plugin-image'

import pkg from './package.json' assert { type: 'json' }

export default {
  input: 'src/index.ts',
  output: [
    {
      file: pkg.main,
      format: 'cjs',
      exports: 'named',
      sourcemap: true,
      strict: false
    },
    {
      file: pkg.module,
      format: 'esm',
      exports: 'named',
      sourcemap: true,
      strict: false
    },
    {
      file: pkg.browser,
      format: 'umd',
      exports: 'named',
      sourcemap: true,
      strict: false,
      name: 'xxx'
    }
  ],
  plugins: [
    image(),
    postcss({
      extract: false,
      modules: true,
      use: ['sass']
    }),
    typescript()
  ],
  external: ['react', 'react-dom']
}

```
package.json
```
{
  "name": "xxx",
  "version": "0.0.1",
  "description": "xxx",
  "main": "./dist/index.cjs.js",
  "module": "./dist/index.esm.js",
  "browser": "./dist/index.umd.js",
  "type": "module",
  "license": "MIT",
  "scripts": {
    "build": "rollup -c",
    "dev": "rollup -c -w",
    "lint": "eslint \"**/*.ts*\""
  },
  "files": [
    "dist"
  ],
  "devDependencies": {
    "@rollup/plugin-image": "^3.0.2",
    "@types/react": "^18.2.0",
    "@types/react-dom": "^18.2.0",
    "eslint": "^7.32.0",
    "eslint-config-custom": "workspace:*",
    "react": "^17.0.2",
    "rollup-plugin-postcss": "^4.0.2",
    "rollup-plugin-typescript2": "^0.34.1",
    "tsconfig": "workspace:*",
    "typescript": "^4.5.2",
    "cesium": "^1.105.1"
  }
}

```
 
Did I do something wrong somewhere？What should i do to solve the problems？

Look forward to your reply ，Thanks!

**Comment by ggetz** [comment]
Hi @wenzhihao123. Thanks for the report. Two questions:

1. Do you create a `Viewer` object before or after setting `Ion.defaultAccessToken`?
2. Is the issue occurring in 1.105.2? See https://github.com/CesiumGS/cesium/issues/11278

**Comment by wenzhihao123** [comment]
> Hi @wenzhihao123. Thanks for the report. Two questions:
> 
> 1. Do you create a `Viewer` object before or after setting `Ion.defaultAccessToken`?
> 2. Is the issue occurring in 1.105.2? See [105.1 Have a Fatal Error on 'https://sandcastle.cesium.com/?src=Cesium Inspector.html' #11278](https://github.com/CesiumGS/cesium/issues/11278)

Thanks for your reply！
1.I create  a `Viewer` object  after setting `Ion.defaultAccessToken`.
2.I update the version to 1.105.2，the error is gone.

Thanks too much, by the way，i also find an error when i use 1.105.1，the code like this:
```
entity.position = new Cesium.ConstantPositionProperty(polyCenter)
entity.label = new Cesium.LabelGraphics({
          text: entity.name,
          fillColor: Cesium.Color.WHITE,
          showBackground: false,
          font: 'normal 18px MicroSoft YaHei',
          distanceDisplayCondition: new Cesium.DistanceDisplayCondition(10000, 900000),
          scaleByDistance: new Cesium.NearFarScalar(50000, 1, 500000, 0.5),
          verticalOrigin: Cesium.VerticalOrigin.BOTTOM,
          style: Cesium.LabelStyle.FILL_AND_OUTLINE,
          outlineWidth: 2,
          outlineColor: Cesium.Color.BLACK,
          heightReference: Cesium.HeightReference.CLAMP_TO_GROUND,
          disableDepthTestDistance: Number.POSITIVE_INFINITY
})
```
the error like this:
```
DeveloperError: Width must be less than or equal to the maximum texture size (0).  Check maximumTextureSize.

Error
    at new DeveloperError (http://localhost:5174/node_modules/.vite/deps/cesium.js?v=74f30f04:7496:11)
    at new Texture (http://localhost:5174/node_modules/.vite/deps/cesium.js?v=74f30f04:28263:11)
    at resizeAtlas (http://localhost:5174/node_modules/.vite/deps/cesium.js?v=74f30f04:102455:29)
    at addImage (http://localhost:5174/node_modules/.vite/deps/cesium.js?v=74f30f04:102533:5)
    at getIndex (http://localhost:5174/node_modules/.vite/deps/cesium.js?v=74f30f04:102543:3)
    at TextureAtlas.addImageSync (http://localhost:5174/node_modules/.vite/deps/cesium.js?v=74f30f04:102563:11)
    at addGlyphToTextureAtlas (http://localhost:5174/node_modules/.vite/deps/cesium.js?v=74f30f04:106385:41)
    at rebindAllGlyphs2 (http://localhost:5174/node_modules/.vite/deps/cesium.js?v=74f30f04:106489:11)
    at LabelCollection.update (http://localhost:5174/node_modules/.vite/deps/cesium.js?v=74f30f04:106846:7)
    at EntityCluster.update (http://localhost:5174/node_modules/.vite/deps/cesium.js?v=74f30f04:128767:27)

```
This problem has been bothering me all day yesterday，but 1.105.2  don't occur. Has 1.105.1 modified the rendering related code？I think we can deprecated the 1.105.1 version to reduce unnecessary trouble.

Thanks again! Best wishes!
.

**Comment by ggetz** [comment]
Thanks, glad it's been fixed! There was a [dependency problem in 1.105.1 which was manifesting as renderer errors](https://github.com/CesiumGS/cesium/issues/11278), despite no rendering changes.
